tutorials/creating_sequences.ipynb

```diff
@@ -296,7 +299,7 @@
     "<img src=\"../files/AF_Ising_program.png\" alt=\"AF Pulse Sequence\" width=\"300\">\n",
     "</center>\n",
     "\n",
-    "The Hamiltonian we are implementing is (the phase is constant and equal to $0$ over time):\n",
+    "The [Hamiltonian](../programming.md#hamiltonian-evolves-state) we are implementing is (the phase is constant and equal to $0$ over time):\n",
```

Comment:
I can tell you are still writing these manually instead of copying :wink: 
```suggestion
    "The [Hamiltonian](../programming.md#hamiltonian-evolves-the-state) we are implementing is (the phase is constant and equal to $0$ over time):\n",
```